src/reports/repo/misc.py

```diff
@@ -1,8 +1,43 @@
 import datetime
 from collections import defaultdict, OrderedDict
-
-import rpmUtils.miscutils
-import yum
+from operator import attrgetter
+import os
+import glob
+import pwd
+import stat
+import tempfile
+
+from .rpmutils import evrcmp
+
+
+# TODO drop this on python3 port
+def to_unicode(astr):
+    if isinstance(astr, unicode):
+        return astr
+    return unicode(astr, 'utf-8', 'replace')
+
+
+def _get_cache_dir(tmpdir='/var/tmp'):
```

Comment:
The orginal took a "yum-" prefix which I think made the path different. Does that matter?
https://github.com/rpm-software-management/yum/blob/master/yum/misc.py#L695

Reply:
Doesn't matter :)